### ISSUE TITLE
Scudcloud crashes on @channel notification

### ISSUE THREAD

**Issue by Alveel** [raelgc/scudcloud#522]
## ScudCloud Version

Paste the output for `scudcloud --version` below:
```ScudCloud 1.40
Python 3.6.0
Qt 5.8.0
PyQt 5.7.1
SIP 4.19
```

## Distro and Desktop info

  - Distribution: ArchLinux
  - Version/Release: Latest
  - Desktop environment: None, using i3 WM.

## Expected behavior

`@channel` notification sends a sound and I maybe get a popup.

## Actual behavior

Scudcloud crashes:
```
alwyn@aristotle ~ % scudcloud
QSocketNotifier: Can only be used with threads started with QThread
QSocketNotifier: Can only be used with threads started with QThread
QPainter::end: Painter not active, aborted
QPainter::end: Painter not active, aborted
QPainter::end: Painter not active, aborted
QPainter::end: Painter not active, aborted
QPainter::end: Painter not active, aborted
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/scudcloud/wrapper.py", line 259, in sendMessage
    self.window.notify(title, str(message), self.icon)
  File "/usr/lib/python3.6/site-packages/scudcloud/scudcloud.py", line 444, in notify
    self.notifier.notify(title, message, icon)
  File "/usr/lib/python3.6/site-packages/scudcloud/notifier.py", line 37, in notify
    notice.show()
GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
[1]    30466 abort (core dumped)  scudcloud
```

## Steps to reproduce
 1. Enable notifications (default Slack settings work)
 2. Have a team member do a `@channel` notification
 3. Scudcloud crashes

## Additional notes
I am using the latest version from the AUR [here](https://aur.archlinux.org/packages/scudcloud/).

I have not been able to get scudcloud to crash with `@here` (yet).

Interestingly enough, the crash never occurs when a normal username mention, either via `alwyn` or `@alwyn`, is triggered.

Chrome, Thunderbird and some other applications seem to have no issues placing notifications.

**Comment by raelgc** [comment]
Wondering if the `@channel` word in the message content (or title) is raising any conflict in DBus.

Anyway, we'll need someone with Arch to help.

**Comment by raelgc** [comment]
I played a bit with the qt5 version today and it happens in Ubuntu too. But it reported a segfault with a non-python stack.

**Comment by raelgc** [comment]
@Alveel Any chance this help you? https://bbs.archlinux.org/viewtopic.php?id=75078

**Comment by raelgc** [comment]
In Ubuntu, qt5 ScudCloud stopped to crash with `@channel` notification after install `libnotify-cil-dev` package.

**Comment by Alveel** [comment]
Reading [this wiki page](https://wiki.archlinux.org/index.php/Desktop_notifications) and after installing dunst, the issue seems to not longer occur, however I think a descriptive error message might help users more :)

Now I had to launch scudcloud via console and log the output before I could figure out what was wrong.

**Comment by raelgc** [comment]
@Alveel I'm glad you found a solution too! Please, raise any other issue you find in qt5 version. Soon I'll promote to be the main version.

What I still unable to make it work: the red marks for bad spelled words.

**Comment by menesis** [comment]
I don't see why `libnotify-cil-dev` (a C# development library) would be needed for a python application. On Ubuntu, that package recommends `notification-daemon` which is provided by `notify-osd`. This is a package that contains `/usr/share/dbus-1/services/org.freedesktop.Notifications.service` that was missing.

Please replace `libnotify-cil-dev` with `notification-daemon`.

**Comment by raelgc** [comment]
@menesis Thank you for investigating. Do you mind to edit the `control` file and open a pull-request for it?

**Comment by raelgc** [comment]
Fixed in commit b0eefa3
